blocks/components/editable/tinymce.js

```diff
@@ -3,7 +3,11 @@ export default class TinyMCE extends wp.element.Component {
 		this.initialize();
 	}
 
-	shouldComponentUpdate() {
+	shouldComponentUpdate( nextProps ) {
+		if ( this.editorNode.getAttribute( 'data-is-empty' ) !== nextProps.isEmpty ) {
```

Comment:
I don't think `shouldComponentUpdate` should be doing any logic aside from deciding if a rerender should occur. Unless I'm mistaken, `componentWillReceiveProps` is still fired even though we return `false` here? Might be the more appropriate lifecycle method.